src/main/resources/static/css/styl1.css

```diff
@@ -0,0 +1,16 @@
+html{
```

Comment:
you should use flex to put elements in correct place:
https://kursownik.pl/kursy/html5_plus_css3/5/1/